{"name": "test_LANDevice", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range", "trace": "self = <test_plugins.Test_plugins object at 0x00000207A4DBCFD0>\n\n    def test_LANDevice(self):\n        name1 = \"LANDevice\"\n        self.p2 = modify_conf_file()\n        self.p2.GetProperitesOfSubObjects()\n        self.p1.send_cmd(\"tftp -g -r testserver.conf 192.168.1.65\")\n        rel = self.p1.send_cmd(\"ls -l\")\n        self.p1.log(rel,name1,self.nowtime)\n        rel=self.p1.send_cmd(\"kill -USR1 `pidof testserver`\")\n        rel=self.p1.send_enter()\n        self.p1.log(rel,name1,self.nowtime)\n        rel = re.findall(r'\"Status\":\"(\\S)\"',rel)\n        print(rel)\n        print(type(rel))\n>       if rel[0] == \"0\":\nE       IndexError: list index out of range\n\ntest_plugins.py:57: IndexError"}, "attachments": [{"name": "log", "source": "715f0eb3-feae-46dd-9b53-209ab9ac90b2-attachment.txt", "type": "text/plain"}], "start": 1700728669833, "stop": 1700728669842, "uuid": "dfa79ae0-f12c-4e0e-943b-e404cc42ee8f", "historyId": "819dd07a0455106f3ec3b53725352a6f", "testCaseId": "819dd07a0455106f3ec3b53725352a6f", "fullName": "test_plugins.Test_plugins#test_LANDevice", "labels": [{"name": "suite", "value": "test_plugins"}, {"name": "subSuite", "value": "Test_plugins"}, {"name": "host", "value": "l29707B"}, {"name": "thread", "value": "26456-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_plugins"}]}